examples/calculator/pom.xml

```diff
@@ -6,7 +6,7 @@
     <description>A simple calculator application</description>
     <groupId>com.spotify</groupId>
     <artifactId>calculator-service</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
Yes, but the cumbersomeness have come from having to exclude it in every mvn release:prepare, and it refusing to keep the snapshot version when asking for what version it should set that artifact to.

I guess we should start separating more of the versioning. But does that mean we should have multiple builds set up, for each part of the project? (api, impl, http module, assembly, examples)